### ISSUE TITLE
'mapper next' recognize clicked 'mapper area/find' hyperlinks

### ISSUE THREAD

**Issue by fiendish** [fiendish/aardwolfclientpackage#170]
_From [ueuecoy...@gmail.com](https://code.google.com/u/102900239435705537225/) on December 22, 2011 22:56:20_

Clicking on a line in 'mapper find' or 'mapper area' should set that room/line as current for purposes of 'mapper next'; currently, 'mapper next' always starts at the first entry on the list.

( r1543 )

_Original issue: http://code.google.com/p/aardwolfclientpackage/issues/detail?id=170_

**Comment by fiendish** [comment]
_From [mip...@gmail.com](https://code.google.com/u/110784188131865699556/) on December 23, 2011 04:08:39_

I would have to agree. I never noticed this before since I did not use this feature that much. There are some rooms I cannot access from the "original" database due to being in the class towers in The Grand City of Aylor. I found this out when I tried to click on hyperlink that was not the first one and typed mapper next. It did nothing since I could not go to the first item in the mapper find or mapper area.
